app/views/external_users/claims/case_details/_transfer_court_fields.html.haml

```diff
@@ -1,14 +1,14 @@
-.form-group.js-typeahead{ class: error_class?(@error_presenter, :transfer_court) ? 'form-group-error dropdown_field_with_errors' : '' }
-  %a#transfer_court
-  %label.form-label-bold{ for: 'claim_transfer_court_id' }
-    = t('.transfer_court')
-    %span.form-hint.xsmall.zero-vert-margin
-      = t('.transfer_court_hint')
-  = validation_error_message(@error_presenter, :transfer_court)
-  = f.collection_select :transfer_court_id, Court.alphabetical, :id, :name, { include_blank: ''.html_safe }, { class: 'form-control fx-autocomplete', 'aria-label': t('.transfer_court') }
+.govuk-form-group.fx-autocomplete-wrapper
+  = f.govuk_collection_select :transfer_court_id,
+    Court.alphabetical,
+    :id,
+    :name,
+    options: { include_blank: '' },
+    label: { text: t('.transfer_court') },
+    hint: { text: t('.transfer_court_hint') }
 
-.form-group.js-typeahead{ class: error_class?(@error_presenter, :transfer_case_number) }
-  = f.adp_text_field :transfer_case_number,
-    label: t('.transfer_case_number_or_urn'),
-    hint_text: t('.transfer_case_number_or_urn_hint'),
-    input_classes: '', errors: @error_presenter
+.govuk-form-group
```

Comment:
helper adds `.govuk-form-group` already. do we need to wrap it up again with the same class?

Reply:
![Screenshot 2022-01-14 at 11 40 38](https://user-images.githubusercontent.com/90189839/149509971-12c04ecc-4b40-4e24-bd0c-116f11e97b31.png)